software/docker/kafka-can-log/depcomp

```diff
@@ -0,0 +1,791 @@
+#! /bin/sh
```

Comment:
I don't think this is supposed to be committed. We should add/update `.gitignore`